azurerm/helpers/azure/app_service.go

```diff
@@ -316,6 +316,17 @@ func SchemaAppServiceSiteConfig() *schema.Schema {
 								Optional:     true,
 								ValidateFunc: validation.StringIsNotEmpty,
 							},
+							"name": {
+								Type:         schema.TypeString,
+								Optional:     true,
+								ValidateFunc: validation.StringIsNotEmpty,
+							},
+							"priority": {
+								Type:         schema.TypeInt,
+								Optional:     true,
+								Computed:     true,
+								ValidateFunc: validation.IntAtLeast(1),
```

Comment:
Should you check here as well if priority is maximum 65000?

Reply:
good call

Reply:
are you sure about the max value being 65000? after some testing, it looks like the max value is 2147483647 for me.

Reply:
good question actually. I just tried to set a rule via az powershell:
`Add-AzWebAppAccessRestrictionRule: Cannot validate argument on parameter 'Priority'. The 65001 argument is greater than the maximum allowed range of 65000. Supply an argument that is less than or equal to 65000 and then try the command again.`

The portal lets you also set something larger than 65000 and enforces the limit at 2147483647. So I guess you are right, that should be it.